dakis/grapesjs-mjml
# [FIX] fixed mj text for erb children

Using `el.innerHTML` on a variable that have erb inside ie: `<mj-text><%= retailer.name %></mj-text>`
Will return  `&lt;%= retailer.name %&gt;` wich will cause an error when minions try to parse the erb.

This pr make sure that if there is the <% %> pattern inside a mj-text, it will use the innerText instead returning the children value without replacing non html value for it's unicode value.

## How to test
 - Checkout out scribe on the branch fix/mj-raw
 - Change `"grapesjs-mjml": "git://github.com/dakis/grapesjs-mjml#master",` for `"grapesjs-mjml": "git://github.com/dakis/grapesjs-mjml#fix/erb-in-mj-text",` in the project dependencies
 - npm install
 - In scribe, drag and drop variables
 - When inspecting the code output (the </> icon) you shouldn't see unicodes
